src/markdown/tutorial/part-2/11-ember-data.md

```diff
@@ -152,12 +151,17 @@ The generator created some boilerplate code for us, which serves as a pretty goo
 
 This model test is also known as a *[unit test](../../../testing/testing-models/)*. Unlike any of the other tests that we've written thus far, this test doesn't actually *render* anything. It just instantiates the rental model object and tests the model object directly, manipulating its attributes and asserting their value.
 
-It is worth pointing out that EmberData provides a `store` *[service](../../../services/)*, also known as the EmberData store. In our test, we used the `this.owner.lookup('service:store')` API to get access to the EmberData store. The store provides a `createRecord` method to instantiate our model object for us.
+It is worth pointing out that EmberData provides a `store` *[service](../../../services/)*, also known as the EmberData store. In our test, we used the `this.owner.lookup('service:store')` API to get access to the EmberData store. The store provides a `createRecord` method to instantiate our model object for us. To make this `store` service available, we must add the following file:
+
+```run:file:create cwd=super-rentals filename=app/services/store.js
+export { default } from 'ember-data/store';
```

Comment:
we might need to disable a lint rule here btw. Been needing to PR fixing the ember-data lint rule to allow `ember-data/store`, it's the one path from that project that folks actually should use.

Reply:
The tutorial builds, so adding this file passes the default lints. The tutorial also fails for deprecations, so with this, there are none being triggered.